13.0/apache/Dockerfile

```diff
@@ -96,6 +96,8 @@ RUN { \
 VOLUME /var/www/html
 
 RUN a2enmod rewrite
+RUN a2enmod remoteip
+RUN { echo RemoteIPHeader X-Real-IP ; echo RemoteIPTrustedProxy 10.0.0.0/8 ; echo RemoteIPTrustedProxy 172.16.0.0/12 ; echo RemoteIPTrustedProxy 192.168.0.0/16 ; } > /etc/apache2/conf-enabled/remoteip.conf; chown www-data:root /etc/apache2/conf-enabled/remoteip.conf
```

Comment:
please reduce the number of layer by merging these three run statements into one

Reply:
Good idea, will do!